### PR TITLE
Sort Pod environment values in alphabetical order

### LINKED ISSUE #739: Sort Environment section of Pod drawer alphabetically
**What would you like to be added**:
When a Pod is selected and the slide out drawer is displayed, it would be useful to have the Environment Variables sorted alphabetically.

**Why is this needed**:
It is sometimes necessary to search for an Environment Variable in the Pod slide-out drawer. In lieu of being able to Ctrl+F in this area [(enhancement request](https://github.com/lensapp/lens/issues/334)), the user is left to read through this list. This experience would be much easier if the list was sorted alphabetically. 

**Environment you are Lens application on:**
3.5.3

- Kubernetes distribution: N/A
- Desktop OS: Mac/OSX

### PR DESCRIPTION
**Pull request by stevejr**
Fixes: #739

Signed-off-by: Steve Richards <srichards@mirantis.com>

### CODE REVIEW

File: src/renderer/components/+workloads-pods/pod-container-env.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { secretsStore } from "../+config-secrets/secrets.store";
 import { configMapsStore } from "../+config-maps/config-maps.store";
 import { Icon } from "../icon";
 import { base64, cssNames } from "../../utils";
+import _ from "lodash";
```
Comment (Nokel81) [inline src/renderer/components/+workloads-pods/pod-container-env.tsx:13]: On line 4 above we already import from lodash, could you change all the uses of `flatten` to `_.flatten` and remove the above?

File: src/renderer/components/+workloads-pods/pod-container-env.tsx [hunk 2]
```diff
@@ -40,7 +41,9 @@ export const ContainerEnvironment = observer((props: Props) => {
   )
 
   const renderEnv = () => {
-    return env.map(variable => {
+    let orderedEnv = _.sortBy(env, 'name');
```
Comment (Nokel81) [inline src/renderer/components/+workloads-pods/pod-container-env.tsx]: nit: use `const`

Comment (stevejr) [reply src/renderer/components/+workloads-pods/pod-container-env.tsx]: Done.

File: src/renderer/components/+workloads-pods/pod-container-env.tsx (hunk 1 above)
Comment (stevejr) [reply src/renderer/components/+workloads-pods/pod-container-env.tsx:13]: Done

Comment (jim-docker) [reply src/renderer/components/+workloads-pods/pod-container-env.tsx:13]: Or couldn't you just import `sortBy` like `flatten`? Seeing `_` is so 2010 :)

**Review by jim-docker — APPROVED** [review]

**Pull request merged**